src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import covidReducer from './continentalCovidData/continentCovid.reducer';
+import countryCovidDataReducer from './countryCovidData/countryCovid.reducer';
+
+const rootReducer = combineReducers({
+  continent: covidReducer,
+  country: countryCovidDataReducer,
+});
+
+const store = createStore(rootReducer, applyMiddleware(logger, thunk));
+
```

Comment:
- [ ] Please Do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.

Reply:
yes